src/rpc/RPCHelpers.hpp

```diff
@@ -557,6 +557,16 @@ parseIssue(boost::json::object const& issue);
 bool
 specifiesCurrentOrClosedLedger(boost::json::object const& request);
 
+/**
+ * @brief Check if a request requires ADMIN role to execute in rippled.
```

Comment:
Maybe something like "Check whether a request is considered to require administrative privileges on `rippled` side" is more accurate? In reality some APIs require IDENTIFIED role, not ADMIN on `rippled` side.